assets/src/components/detours/diversionPage.tsx

```diff
@@ -12,7 +13,15 @@ export const DiversionPage = ({
   routeDirection,
   routeOrigin,
   shape,
-}: DiversionPanelProps & { shape: Shape }) => {
+  onClose,
+  center,
+  zoom,
+}: DiversionPanelProps & {
```

Comment:
I think this needs a separate type. There's some overlap between `DiversionPanelProps` and ... effectively `DiversionPageProps`, but less and less - for instance, `directions` are part of `DiversionPanelProps`, but `DiversionPage` (correctly) ignores them in favor of the directions it gets from `useDetour()`.